doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,182 @@
+==================================
+Small Scale Rotating Drum (3d)
+==================================
+
+This example of Lethe-DEM is to simulate the behaviour of dry granular flow in a small scale rotating drum. The discrete element method DEM will be responsible of describing the behaviour of particles. DEM parameters are described in Lethe-DEM documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Rotational boundary
+- Load-balancing
+
+
+Location of the examples
+------------------------
+ ``/examples/dem/small_scale_rotating_drum/small_rotating_drum_dem.prm``
+
+
+Description of the case
+-----------------------
+
+This example simulates a cascading regime in a small scale rotating drum. First, we first use Lethe-DEM to fill the bed with 20000 particles. We enable check-pointing in order to write the DEM checkpoint files which will be used as the starting point of the DEM simulations . Then, the solver ``dem_3d`` is used to simulate the behaviour of dry granular flow within the rotating drum.
```

Comment:
I don't understand this sentence: 
`We enable check-pointing in order to write the DEM checkpoint files which will be used as the starting point of the DEM simulations .`
You enable checkpointing  to be able to restart the DEM simulation? Or do you somehow do a simulation before to obtain files that are used to start a new simulation?